### ISSUE TITLE
Requesting extra information for `status` command when we use `-r and -s` options

### ISSUE THREAD

**Issue by savitaashture** [cloudfoundry-incubator/lattice#112]
The new Status command looks very detailed with CPU,Memory and Disk %.  We would like to get the same when `-r and -s` are used.  This helps us point out a particular instance which is taking the load.

Current Output:
```
================================================================================
Instance        State           Crashes  Since
--------------------------------------------------------------------------------
0       RUNNING      0          2015-04-27 21:07:54 (EDT)
1       RUNNING      0          2015-04-27 22:06:12 (EDT)
2       RUNNING      0          2015-04-27 22:06:12 (EDT)
```

Requested Output:

```
====================================================================================================
Instance        State           Crashes  Since          CPUPercentage   MemoryUsage     DiskUsage
----------------------------------------------------------------------------------------------------
0       RUNNING      0          2015-04-27 23:10:04 (EDT)       41.66           852K            0
1       RUNNING      0          2015-04-27 23:12:27 (EDT)       0.01            444K            0
2       RUNNING      0          2015-04-27 23:12:21 (EDT)       41.65           1020K           0
3       RUNNING      0          2015-04-27 23:12:27 (EDT)       0.01            440K            0
4       RUNNING      0          2015-04-27 23:12:26 (EDT)       0.01            440K            0
```

Would you like a PR for this change?

**Comment by davidwadden** [comment]
That sounds like a good idea.  I had sort of planned to do this at some point, but I'd be happy to consider a PR to add this functionality.  Couple thoughts:

* Try to minimize the overall width of the instance summary output
  - Change Since to be Uptime, calculate duration elapsed since `Since`, e.g., 16m, 1d15h27m; maybe using the following:
    - http://golang.org/pkg/time/#Since
    - http://golang.org/pkg/time/#Duration.String
* Feel free to change padding in tabwriter (can be a bit quirky)
* Change header from **CPUPercentgage** to **CPU**, append **%** to values.
  - Can do the same on non-summary status output

Also, the values of disk usage will show as zero until we enable disk quotas in garden-linux.  In order to see for yourself:
1. `vagrant ssh` in
2. `sudo su`
3. Remove line 14 `-disableQuotas=true \` from `/etc/init/garden-linux.conf` 
4. Run (as root): `stop garden-linux;  start garden-linux`

We're trying to bring in disk quotas soon, but need to do some more testing.

**Comment by savitaashture** [comment]
Hi @davidwadden 
Thanks for the feedback. :-) will change accordingly and send the PR.

**Comment by davidwadden** [comment]
heads up @savitaashture 

we decided to keep disk quotas disabled, and we're going to remove displaying `Disk Usage` on the `ltc status` output.

**Comment by menicosia** [comment]
This has been addressed in PR #115, closing. As always, please feel free to re-open if you feel there is something that should not be lost by closing this ticket!

--
  Marco Nicosia
  Product Manager
  Pivotal Software, Inc.
